pkg/services/sqlstore/migrations/migrations.go

```diff
@@ -87,6 +87,12 @@ func (*OSSMigrations) AddMigration(mg *Migrator) {
 			addCommentMigrations(mg)
 		}
 	}
+
+	if mg.Cfg != nil && mg.Cfg.IsFeatureToggleEnabled != nil {
+		if mg.Cfg.IsFeatureToggleEnabled(featuremgmt.FlagStorage) {
+			addDbFileStorageMigration(mg)
+		}
+	}
```

Comment:
Swooping in from the HG team here - I'd really recommend that any schema changes are not put behind a feature flag. We want schema changes to be as reliable and boring as possible. If the migration is only making additive changes, there is no drawback to performing the migration before the feature is 'enabled'.

Reply:
I appreciate that there are examples of us doing exactly the same in the lines above this, but its never too late to start a good practice!

Reply:
hey @dafydd-t thanks for the drive by 🏎! it does make sense to me, since we shouldn't change migrations that are committed to main anyway:
https://github.com/grafana/grafana/blob/main/pkg/services/sqlstore/migrations/migrations.go#L12-L15

thoughts @ryantxu ?

Reply:
@dafydd-t -- agree, with the caveat that the schema migration can change while the feature requires dev mode :)

When we remove dev-mode, the migration should happen for everyone

Reply:
`storage` flag isn't dev-mode only, so I removed the condition in this PR 🍾